Chromatography/CZE.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Chromatography
+{
+    public class CZE
+    {
+        public double ExperimentalElectrophoreticMobility(double length, double timeMin, double voltsPerMeter)
+        {
+            if (length >= 0 && timeMin >= 0)
+                return (length / (60 * timeMin * voltsPerMeter) * 1e9);
+            else
+                return (-1);
+        }
+
+        public double PredictedElectrophoreticMobility(string peptideSequence, double observedMass)
+        {
+            double predictedMu = 0;
+
+            predictedMu = 3.069 + 386 * Math.Log(1d + 0.35 * PredictedChargeCorrected(peptideSequence)) / (Math.Pow(observedMass, 0.411) + Offset(PredictedChargeCorrected(peptideSequence), peptideSequence.Length));
+            return predictedMu;
+        }
+
+        private double PredictedCharge(string peptideSequence)
```

Comment:
Consider making this return `int` and removing the `double` casting. You're adding it to a `double` below, which is fine.